lib/src/ast/sass/expression/function.dart

```diff
@@ -24,39 +25,39 @@ class FunctionExpression
   /// without a namespace.
   final String? namespace;
 
-  /// The name of the function being invoked.
-  ///
-  /// If [namespace] is non-`null`, underscores are converted to hyphens in this name.
-  /// If [namespace] is `null`, this could be a plain CSS function call, so underscores are kept unchanged.
+  /// The name of the function being invoked, with underscores left as-is.
+  final String originalName;
+
+  /// The name of the function being invoked, with underscores converted to
+  /// hyphens.
   ///
-  /// If this is interpolated, the function will be interpreted as plain CSS,
-  /// even if it has the same name as a Sass function.
-  final Interpolation interpolatedName;
-  String? get name => namespace == null
-      ? interpolatedName.asPlain?.replaceAll('_', '-')
-      : interpolatedName.asPlain;
+  /// If this function is a plain CSS function, use [originalName] instead.
+  final String name;
```

Comment:
I'd make this a getter, since a single `String.replaceAll()` isn't very expensive.